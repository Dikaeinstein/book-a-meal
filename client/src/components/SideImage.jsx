import React from 'react';
import PropTypes from 'prop-types';

const SideImage = ({ id }) => (
  <section className="col-3-4 hero text-center" id={id}>
    <div>
      <img
        src="http://res.cloudinary.com/dikaeinstein/image/upload/c_scale,q_auto:eco,w_1029/v1525566673/book-a-meal/avocado-cooked-delicious-262959.jpg"
        alt="meal"
      />
    </div>
  </section>
);

SideImage.propTypes = {
  id: PropTypes.string.isRequired,
};

export default SideImage;
