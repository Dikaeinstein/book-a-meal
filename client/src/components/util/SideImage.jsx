import React from 'react';
import PropTypes from 'prop-types';

const SideImage = ({ id }) => (
  <section className="col-3-4 hero text-center bg-dark" id={id}>
    <div>
      <img
        src="http://res.cloudinary.com/dikaeinstein/image/upload/c_scale,q_auto:low,w_1077/v1525566660/book-a-meal/close-up-cooking-dinner-46239.jpg"
        alt="meal"
      />
    </div>
  </section>
);

SideImage.propTypes = {
  id: PropTypes.string.isRequired,
};

export default SideImage;
