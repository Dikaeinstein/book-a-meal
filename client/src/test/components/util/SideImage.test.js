import React from 'react';
import { shallow } from 'enzyme';
import SideImage from '../../../components/util/SideImage';

describe('<SideImage />', () => {
  it('should match snapshot', () => {
    const tree = shallow(<SideImage id="testIn" />);
    expect(tree).toMatchSnapshot();
  });
});
