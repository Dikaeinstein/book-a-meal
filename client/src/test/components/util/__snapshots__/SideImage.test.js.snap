// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideImage /> should match snapshot 1`] = `
<section
  className="col-3-4 hero text-center bg-dark"
  id="testIn"
>
  <div>
    <img
      alt="meal"
      src="http://res.cloudinary.com/dikaeinstein/image/upload/c_scale,q_auto:low,w_1077/v1525566660/book-a-meal/close-up-cooking-dinner-46239.jpg"
    />
  </div>
</section>
`;
